config/sync/simple_block.simple_block.anonymous_comment_links.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 04adf1f1-d724-4489-8d03-46716e4e3c50
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.content_editor
+id: anonymous_comment_links
+title: 'Anonymous comment links'
+content:
+  value: '<a href="/caslogin">Login</a> or <a href="/user/register">create an account</a> to comment.'
+  format: content_editor
```

Comment:
The wording and URIs here are subtly different from the ones shown inside the comments themselves.

I'm wondering if it is not overkill to repeat this so often, probably this block is enough and we don't need to show the links for each comment.

edit: This is a functional decision, so I posted this question in the functional ticket.